lib/api/protocol/cookie.js

```diff
@@ -12,6 +12,7 @@ const ProtocolAction = require('./_base-action.js');
  * @see deleteCookie
  * @see deleteCookies
  * @api protocol.cookies
+ * @deprecated
```

Comment:
add some links here

Reply:
All the cookies commands are its replacement actually, since it's a low-level command which Nightwatch itself uses.

Reply:
okay